backend/app/features/schema_editor/routers.py

```
@@ -81,6 +81,21 @@ def update_column(
     return ColumnMutationResponse(**service.update_column(table_id, column_id, payload, ctx))
 
 
+@router.delete(
+    "/diagrams/{diagram_id}/tables/{table_id}/columns/{column_id}",
+    response_model=ColumnMutationResponse,
+)
+def delete_column(
+    diagram_id: str,
+    table_id: str,
+    column_id: str,
+    ctx: RequestContext = Depends(get_request_context),
```

Ruff (0.15.2)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)